API.php

```diff
@@ -1448,7 +1448,7 @@ public function exportContainerVersion($idSite, $idContainer, $idContainerVersio
      * @param string $backupName   If specified, a backup of the current draft will be created under this version name.
      * @return array
      */
-    public function importContainerVersion($exportedContainerVersion, $idSite, $idContainer, $backupName = '')
+    public function importContainerVersion($exportedContainerVersion, $idSite, $idContainer, $backupName = '', $isDraftRestoreCall = false)
```

Comment:
I came across this change due to a UI screenshot being changed in core:

Is this parameter supposed to be public API? If so it would be helpful to clearly describe its purpose at least in the doc block. By just looking at the parameter name I don't have a clue what it's supposed to do.
If it's more like an internal parameter, only used by our UI, that shouldn't be used by API consumers directly, it should be named `$_isDraftRestoreCall`, as that would hide it from the API listing page.

Also it would be awesome if you could start adding type hints for all API parameters where the type can be hinted. At least for new parameters added.

Reply:
@sgiehl thanks for the info, created - https://github.com/matomo-org/tag-manager/pull/1038